_data/simple-icons.json

```
@@ -11976,6 +11976,11 @@
             "hex": "F5CD0E",
             "source": "https://www.cloudskillsboost.google"
         },
+        {
+            "title": "Qwik",
+            "hex": "AC7EF4",
+            "source": "https://qwik.builder.io/"
+        },
```

```suggestion
        {
            "title": "Qwik",
            "hex": "AC7EF4",
            "source": "https://qwik.builder.io"
        },
```

If you apply this change, and move it above Qwiklabs in the JSON, the lint error should resolve.